google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceLiveTest.java

```diff
@@ -67,6 +74,34 @@ public void testListHardwareProfiles() throws Exception {
       }
    }
 
+   public void testCreateNodeWithSsd() throws Exception {
+      String group = this.group + "ssd";
+      try {
+         TemplateOptions options = client.templateOptions();
+
+         options.as(GoogleComputeEngineTemplateOptions.class).bootDiskType("pd-ssd");
```

Comment:
Move this string (and the other possible values) to a constant somewhere so it can be reused by the users?